src/__tests__/error-boundary.test.tsx

```diff
@@ -0,0 +1,30 @@
+import { render, screen } from '@testing-library/react';
+import { ErrorBoundary } from '../components/error-boundary/error-boundary';
+import { messages } from '../sources/messages';
+import { SimulateError } from '../components/simulate-error';
+import userEvent from '@testing-library/user-event';
+
+describe('Error boundary', () => {
```

Comment:
mocking concole.error for nice output
```suggestion
describe('Error boundary', () => {
  const originalConsoleError = globalThis.console.error;
  beforeAll(() => {
    globalThis.console.error = vi.fn();
  });

  afterAll(() => {
    globalThis.console.error = originalConsoleError;
  });

```